### ISSUE TITLE
Quantification

### ISSUE THREAD

**Issue by AmosG8** [jovo/fast-oopsi#7]
First of all, thanks for sharing your codes and for supporting it!
I'm wondering how to convert the values I get from oopsi to 'relative firing rate'.
Do the vertical lines indicate spike rate or do they indicate how certain I am that a single spike has occurred. Of course the two parameters are linked, because there is higher certainty that a neuron has fired when it fires a train as appose to when it fires a single spike.  
In other words, how to convert the result from oopsi to a relative number of spikes.
Thanks for you help!

**Comment by jovo** [comment]
sorry for delay, see answer here: https://github.com/jovo/oopsi/blob/master/README

also, the most up to date code is in the oopsi (not fast-oopsi) repo.

let me know if you have any further questions!
